.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
-	url = https://github.com/celo-org/forge-std
```

Comment:
Does this work ? I can see solidity files that are supporting versions only starting 0.6. 

eg. src/StdError.sol
>=0.6.2 <0.9.0

Are we able to compile and use this in monorepo ?

Reply:
It does, since this is only used to compile contracts with ^0.8.0 versions only.